easybuild/easyconfigs/i/ITK/ITK-5.0.1-foss-2019b-Python-3.7.4.eb

```
@@ -60,6 +60,7 @@ configopts += '-DITK_USE_SYSTEM_ZLIB=ON -DZLIB_INCLUDE_DIR=$EBROOTZLIB/include '
 configopts += '-DZLIB_LIBRARY=$EBROOTZLIB/lib/libz.%s ' % SHLIB_EXT
 configopts += '-DModule_ITKReview=ON -DModule_ITKVtkGlue=ON '
 
+prebuildopts = "env LC_ALL=C "
```

```suggestion
prebuildopts = "LC_ALL=C "
```
is enough (click to apply). Is there any advantage in using the current version?

---

Honestly, I just copied that from another `.eb` using that approch. There shouldn't be an advantage, you're right! Thanks!